customer-service-tests/EmployeeModelTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Xunit;
+using System.Collections.Generic;
+using customer_service;
+using customer_service.Models;
+using customer_service.Data;
+
+namespace customer_service_tests
+{
+    public class EmployeeModelTests
+    {
+        [Fact]
+        public void CanSaveNewEmployeeToDatabase()
+        {
+            var employeeFactory = new EmployeeFactory();
+            List<Employee> allEmployees = employeeFactory.getAll();
+            int newEmployeeId = allEmployees[allEmployees.Count - 1].EmployeeId;
```

Comment:
Hmm, this pattern won't work unless you JUST added a Jeb to your database. This grabs the EmployeeId for the last Employee in the system, before you saved the new Employee. See my suggestion below